tasks/sep/zora-002-fp-upgrade/NestedSignFromJson.s.sol

```diff
@@ -0,0 +1,561 @@
+import {IDelayedWETH} from "@eth-optimism-bedrock/interfaces/dispute/IDelayedWETH.sol";
+import {IOptimismPortal2} from "@eth-optimism-bedrock/interfaces/L1/IOptimismPortal2.sol";
+import {ISystemConfig} from "@eth-optimism-bedrock/interfaces/L1/ISystemConfig.sol";
+import {ISuperchainConfig} from "@eth-optimism-bedrock/interfaces/L1/ISuperchainConfig.sol";
+import {IL1CrossDomainMessenger} from "@eth-optimism-bedrock/interfaces/L1/IL1CrossDomainMessenger.sol";
+import {IL1StandardBridge} from "@eth-optimism-bedrock/interfaces/L1/IL1StandardBridge.sol";
+import {IL1ERC721Bridge} from "@eth-optimism-bedrock/interfaces/L1/IL1ERC721Bridge.sol";
+import {IAddressManager} from "@eth-optimism-bedrock/interfaces/legacy/IAddressManager.sol";
+import {IL1ChugSplashProxy} from "@eth-optimism-bedrock/interfaces/legacy/IL1ChugSplashProxy.sol";
+import {IOptimismMintableERC20Factory} from
+    "@eth-optimism-bedrock/interfaces/universal/IOptimismMintableERC20Factory.sol";
+
+interface ISystemConfigLegacy is ISystemConfig {
+    function l2OutputOracle() external view returns (address);
+}
+
+contract NestedSignFromJson is OriginalNestedSignFromJson, CouncilFoundationNestedSign, SuperchainRegistry {
+    /// @notice Expected address for the AnchorStateRegistry proxy.
+    IAnchorStateRegistry expectedAnchorStateRegistryProxy =
+        IAnchorStateRegistry(vm.envAddress("EXPECTED_ANCHOR_STATE_REGISTRY_PROXY"));
+
+    /// @notice Expected address for the AnchorStateRegistry implementation.
+    IAnchorStateRegistry expectedAnchorStateRegistryImpl =
+        IAnchorStateRegistry(vm.envAddress("EXPECTED_ANCHOR_STATE_REGISTRY_IMPL"));
+
+    /// @notice OP Sepolia address for the AnchorStateRegistry implementation for comparison.
+    IAnchorStateRegistry comparisonAnchorStateRegistryImpl =
+        IAnchorStateRegistry(vm.envAddress("COMPARISON_ANCHOR_STATE_REGISTRY_IMPL"));
+
+    /// @notice Expected address for the PermissionedDisputeGame implementation.
+    IPermissionedDisputeGame expectedPermissionedDisputeGameImpl =
+        IPermissionedDisputeGame(vm.envAddress("EXPECTED_PERMISSIONED_DISPUTE_GAME_IMPL"));
+
+    /// @notice OP Sepolia address for the PermissionedDisputeGame implementation for comparison.
+    IPermissionedDisputeGame comparisonPermissionedDisputeGameImpl =
+        IPermissionedDisputeGame(vm.envAddress("COMPARISON_PERMISSIONED_DISPUTE_GAME_IMPL"));
+
+    /// @notice Expected address for the DelayedWETH proxy.
+    IDelayedWETH expectedDelayedWETHProxy =
+        IDelayedWETH(payable(vm.envAddress("EXPECTED_PERMISSIONED_DELAYED_WETH_PROXY")));
+
+    /// @notice Expected address for the DisputeGameFactory proxy.
+    IDisputeGameFactory expectedDisputeGameFactoryProxy =
+        IDisputeGameFactory(vm.envAddress("EXPECTED_DISPUTE_GAME_FACTORY_PROXY"));
+
+    /// @notice Script constructor.
+    constructor() SuperchainRegistry("sepolia", "zora", "v1.8.0-rc.4") {}
```

Comment:
this should work now that the SR was updated
```suggestion
    constructor() SuperchainRegistry("sepolia", "zora", "v1.8.0") {}
```

Reply:
Requires updated version of the upstream, I didn't want to change that as part of this PR